lib/form/paragraph3.flow

```diff
@@ -114,7 +114,11 @@ makeAlphabetAwareTextField(txt : string, style : [CharacterStyle]) -> native {
 	overridden = applyAlphabetStyles(detectAlphabet(txt), style);
 	textfield = makeTextfield(getMappedFontFamily(overridden));
 	dstyle = getDefinedTextStyle(overridden);
-	setTextAndStyle(textfield, txt, dstyle.face.family, dstyle.fontSize, dstyle.face.weight, dstyle.face.slope, dstyle.fillColor, dstyle.fillOpacity, dstyle.letterSpacing, dstyle.backgroundColor, dstyle.backgroundOpacity);
+	text = if (isUrlParameter753True()) {
+		esc = extractStruct(style, EscapeHTML(true)).escape;
```

Comment:
if it is for CPP only then move extractStruct under cpp to avoid extra call in say JS